components/log_archive/main.tf

```diff
@@ -0,0 +1,27 @@
+variable "name" {
+  description = "The name for the archive bucket (e.g. dosomething-papertrail)."
+}
+
+resource "aws_s3_bucket" "archive" {
+  bucket = var.name
+  acl    = "private"
+
+  tags = {
+    Application = "papertrail"
+    Environment = "all"
+    Stack       = "all"
```

Comment:
I wasn't sure what to put here – since Papertrail is sort-of a "catch all". Open to suggestions!